cmd/zetacored/root.go

```diff
@@ -30,31 +32,48 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/crisis"
 	genutilcli "github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	cosmosevmcmd "github.com/cosmos/evm/client"
+	"github.com/cosmos/evm/crypto/hd"
+	evmdconfig "github.com/cosmos/evm/evmd/cmd/evmd/config"
+	"github.com/cosmos/evm/evmd/eips"
+	cosmosevmserverconfig "github.com/cosmos/evm/server/config"
+	evmtypes "github.com/cosmos/evm/x/vm/types"
+	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/spf13/cast"
 	"github.com/spf13/cobra"
-	ethermintclient "github.com/zeta-chain/ethermint/client"
-	"github.com/zeta-chain/ethermint/crypto/hd"
-	evmenc "github.com/zeta-chain/ethermint/encoding"
-	"github.com/zeta-chain/ethermint/types"
 
 	"github.com/zeta-chain/node/app"
 	zetacoredconfig "github.com/zeta-chain/node/cmd/zetacored/config"
+	"github.com/zeta-chain/node/pkg/chains"
 	zetamempool "github.com/zeta-chain/node/pkg/mempool"
 	zevmserver "github.com/zeta-chain/node/server"
-	servercfg "github.com/zeta-chain/node/server/config"
+	zetaserverconfig "github.com/zeta-chain/node/server/config"
 )
 
 const EnvPrefix = "zetacore"
 
-// NewRootCmd creates a new root command for wasmd. It is called once in the
+// NewRootCmd creates a new root command for zetacored. It is called once in the
 // main function.
 
 type emptyAppOptions struct{}
 
 func (ao emptyAppOptions) Get(_ string) interface{} { return nil }
 
-func NewRootCmd() (*cobra.Command, types.EncodingConfig) {
-	encodingConfig := app.MakeEncodingConfig()
+func NewRootCmd() *cobra.Command {
+	// need to create this app instance to get autocliopts
+	tempApp := app.New(
+		log.NewNopLogger(),
+		dbm.NewMemDB(),
```

Comment:
Is memdb here okay ?

Reply:
yea same on develop, this is just temp app instance